src/app/transaction-spending-limit/transaction-spending-limit-lockup/transaction-spending-limit-lockup.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { GlobalVarsService } from 'src/app/global-vars.service';
+import {
+  LockupLimitMapItem,
+  LockupLimitOperationString,
+  LockupLimitScopeType,
+  User,
+} from '../../backend-api.service';
+import { TransactionSpendingLimitComponent } from '../transaction-spending-limit.component';
+
+@Component({
+  selector: 'app-transaction-spending-limit-lockup',
+  templateUrl: './transaction-spending-limit-lockup.component.html',
+  styleUrls: ['./transaction-spending-limit-lockup.component.scss'],
+})
+export class TransactionSpendingLimitLockupComponent implements OnInit {
+  @Input() lockupLimitMapItem: LockupLimitMapItem | undefined;
+  @Input() appUser: User | undefined;
+  TransactionSpendingLimitComponent = TransactionSpendingLimitComponent;
+  LockupLimitScopeType = LockupLimitScopeType;
+
+  constructor(public globalVars: GlobalVarsService) {}
+
+  ngOnInit(): void {}
+
+  getOperationString(): string {
+    switch (this.lockupLimitMapItem?.Operation) {
+      case LockupLimitOperationString.ANY:
+        return 'Any';
+      case LockupLimitOperationString.COIN_LOCKUP:
+        return 'Lockup';
+      case LockupLimitOperationString.COIN_LOCKUP_TRANSFER:
+        return 'Transfer';
```

Comment:
Just test to make sure these strings look good, make sense, and are sufficiently descriptive wherever they appear.